README.md

```diff
@@ -3,22 +3,50 @@
 [![Build Status](https://travis-ci.org/brettminnie/breakfast-serializer.svg)](https://travis-ci.org/brettminnie/breakfast-serializer)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/brettminnie/breakfast-serializer/badges/quality-score.png?b=develop)](https://scrutinizer-ci.com/g/brettminnie/breakfast-serializer/?branch=develop)
 [![Dependency Status](https://www.versioneye.com/user/projects/55378b007f43bcd88900033d/badge.svg?style=flat)](https://www.versioneye.com/user/projects/55378b007f43bcd88900033d)
+[![Coverage Status](https://coveralls.io/repos/brettminnie/breakfast-serializer/badge.svg?branch=master&service=github)](https://coveralls.io/github/brettminnie/breakfast-serializer?branch=master)
 
-A (hopefully) backwards compatible replacement for the other well known serializer
+A replacement for the other well known serializer, initially we where aiming for backwards compatibility, however it seemed
```

Comment:
:+1: Fixed and clarified end of sentence

On Tue, Jul 21, 2015 at 12:01 PM, François Pietka <notifications@github.com>
wrote:

> In README.md
> <https://github.com/brettminnie/breakfast-serializer/pull/22#discussion_r35090936>
> :
>
> >
> > -A (hopefully) backwards compatible replacement for the other well known serializer
> > +A replacement for the other well known serializer, initially we where aiming for backwards compatibility, however it seemed
>
> we were aiming for
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brettminnie/breakfast-serializer/pull/22/files#r35090936>
> .
>